src/less/tachyons-utils.css

```diff
@@ -0,0 +1,37 @@
+.pg-book {font-family: "PostGrotesk-Book", sans-serif}
```

Comment:
could be worth linking to tachyons in the readme (or somewhere in `/doc`) as it looks like it is growing up beyond being just a helper library and part of the core styling of this repository? in service of https://github.com/status-im/ideas/tree/master/ideas/140-sob-improve-onboarding hopefully 🙏